api/bibliotheca.py

```diff
@@ -1447,9 +1501,44 @@ def handle_event(self, bibliotheca_id, isbn, foreign_patron_id,
                 0, 1
             )
         title = edition.title or "[no title]"
-        self.log.info("%r %s: %s", start_time, title, internal_event_type)
+        self.log.info("%s %s: %s", start_time.strftime(self.LOG_DATE_FORMAT),
+                      title, internal_event_type)
         return start_time
 
+
+class RunBibliothecaMonitorScript(RunCollectionMonitorScript):
+
+    def __init__(self, monitor_class, _db=None, cmd_args=None, **kwargs):
+        super(RunBibliothecaMonitorScript, self).__init__(monitor_class, _db, cmd_args, **kwargs)
+
+    # TODO: Both '--default-start-date' and '--force-default' (or equivalents) should probably end up
+    #   in a more generalized script runner superclass that is focused on Monitor or TimelineMonitor.
```

Comment:
I'm not sure about this; the Bibliotheca monitor is the only one that has needed a default start date. The other ones use the time of the first run as the start date, since they don't need to ask about the past. And once the new Bibliotheca API comes out I think even Bibliotheca won't need this.

Reply:
There RBdigital, which I know is gone, is another case where I would have found that useful and I suspect that there will be cases in the future where it would be nice to get a do-over without having to manually change the database. But I certainly don't mind removing this.

Reply:
It's just a TODO, and who knows what the future will hold.